test/html/test_html.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from fpdf import FPDF, HTMLMixin
```

Comment:
`HTMLMixin` is also used in other unit tests: `test/outline/test_outline_html.py` & `test/test_links.py`.
It is also mentioned in `tutorial/tuto6.p`